compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1290,34 +1298,34 @@ extern "C" {
     pub fn LLVMBuildResume<'a>(B: &Builder<'a>, Exn: &'a Value) -> &'a Value;
     pub fn LLVMBuildUnreachable<'a>(B: &Builder<'a>) -> &'a Value;
 
-    pub fn LLVMRustBuildCleanupPad<'a>(
+    pub fn LLVMBuildCleanupPad<'a>(
         B: &Builder<'a>,
         ParentPad: Option<&'a Value>,
-        ArgCnt: c_uint,
         Args: *const &'a Value,
+        NumArgs: c_uint,
         Name: *const c_char,
     ) -> Option<&'a Value>;
```

Comment:
Sometimes llvm type `LLVMValueRef` modeled in rust as `Option<&'a Value>`, sometimes as `&'a Value`, where `LLVMValueRef` can be nullptr:
https://github.com/llvm/llvm-project/blame/42d1b276f7793999be3f9b6a99efbb143254c729/llvm/lib/IR/Core.cpp#L3170-L3178

It feels wrong and can lead to funny results, or i'm wrong? Don't have strong c++ here, so advice requested.